examples/ensemble/plot_gradient_boosting_regularization.py

```diff
@@ -32,17 +32,17 @@
 from sklearn import datasets
 
 
-X, y = datasets.make_hastie_10_2(n_samples=12000, random_state=1)
+X, y = datasets.make_hastie_10_2(n_samples=3000, random_state=1)
```

Comment:
What do you think of setting the following settings?

```python
from sklearn.model_selection import train_test_split

X, y = datasets.make_hastie_10_2(n_samples=4000, random_state=1)
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.8, random_state=0)

original_params = {
    "n_estimators": 400,
    ...
}
```

It looks like it keeps a very similar message as the original: 

![Figure_1](https://user-images.githubusercontent.com/5402633/143163365-9a33651f-1d4e-4d2a-bc00-06b14d431bdd.png)